### ISSUE TITLE
Replace numbers with #define constants

### ISSUE THREAD

**Issue by CrasherCourse** [teslaworksumn/HeadMaster#23]
I feel using data abstractions in place of certain numbers (eg use NUMBER_OF_SLAVES instead of 4) will help clarify meanings of those numbers and allow for easier changes to those values if needed.

**Comment by taylortrimble** [comment]
Right you are. I'm going to assign this to you; let me know if I shouldn't :stuck_out_tongue_winking_eye:

**Comment by CrasherCourse** [comment]
Cool, I'll do that later today.  For now I need some sleep

Oh! Where should I put the #define section?
After pragmas?

**Comment by taylortrimble** [comment]
I hear ya. I'm going to bed too. Night!

**Comment by taylortrimble** [comment]
You can put the `#define`s after the `#include`s.

``` C
// 
// source.c
// Header
// 

#include "MyLib.h"
#include <xc.h>

// Defines

#define MAGIC 9
#define ANSWER 42

...
```

**Comment by taylortrimble** [comment]
Is this still going?

**Comment by taylortrimble** [comment]
I saw some more of this inside `I2C.c`. We had the number of bytes sent flexible at one point, but that must have been lost inside some complicated merging.

Here's the commit that fixed it once upon a time: ee75060233b3d5b1230b38a00d49b444345c940b

**Comment by taylortrimble** [comment]
I think we have a few more magic numbers to take care of before this is closed; I know we have a magic 4 in here too.
